remoteappmanager/static/js/home/components/application_view_component.js

```diff
@@ -0,0 +1,110 @@
+define([
+    'analytics',
+    'jsapi/v1/resources',
+    './application_list_component',
+    '../../urlutils',
+    '../../utils',
+    '../../dialogs'
+], function(
+        analytics, resources, application_list_component, urlutils, utils, dialogs
+    ) {
+    'use strict';
+
+    var ga = analytics.init();
+
+    var Status = application_list_component.Status;
+
+    var getWindowResolution = function() {
+        var max_size = utils.max_iframe_size();
+        return max_size[0]+"x"+max_size[1];
+    };
+
+    // Ember component for the application VNC view
+
+    var ApplicationViewComponent = Ember.Component.extend({
```

Comment:
The Ember component representing the vnc view of the current clicked application.